### PR TITLE
Align Sig Validity Indicator (#1660)

### LINKED ISSUE #1660: Align sig validity indicator
**Background:**

In the toolbox -> verify section we have the following display:

image

The icon indicates red when invalid and green when valid. At this point . it is completely mis-aligned with the way we display indicators, e.g. see the address icon in the dropdown just above. In all cases the indicators/fields are aligned such that the icon is half in-out the box and the text never moves.

(The same can be seen on the settings page)

**Actions:**

1. Move the display icon to align with the icons on the left in other inputs - very similar to what we have in the actual settings where the icon is to the left of the box, half-overlapping. For inspiration we can take a look at the way it is done there.

2. Adjust the size of these icons, once moved, to be the same size as the other half in-out indicators. (same size as the icons in settings, same size as the icon for accounts)

### PR DESCRIPTION
**Pull request by touhonoob**
resolve #1660

### CODE REVIEW

File: packages/app-toolbox/src/Verify.tsx [hunk 1]
```diff
@@ -64,6 +65,18 @@ class Verify extends React.PureComponent<Props, State> {
   public render (): React.ReactNode {
     const { t } = this.props;
     const { cryptoOptions, cryptoType, data, defaultPublicKey, isHexData, isValid, isValidAddress, isValidSignature, signature } = this.state;
+    const AlignedIcon = styled(Icon)`
```
[the remark this answers is not in the document]
Comment (touhonoob) [reply packages/app-toolbox/src/Verify.tsx]: Hi @jacogr 
Thanks for the review:
1. moved to file-level
2. it's already adjusted to 32px by setting font-size as 24pt
Screenshot from 2019-10-07 22-29-15

File: packages/app-toolbox/src/Verify.tsx [hunk 2]
```diff
@@ -206,4 +208,17 @@ class Verify extends React.PureComponent<Props, State> {
...
+        position: relative;
+        left: 0.75rem;
+        top: 1rem; 
+        width: 32px;
+        height: 32px;
+        font-size: 24pt;
+        background: white !important;
+        border-radius: 50%;
+      }
+    `;
```
Comment (jacogr) [inline packages/app-toolbox/src/Verify.tsx]: Could we indent to 2 speces max, i.e.

```js
const AlignedIcon = styled(Icon)`
  &&&.ui--AlignedIcon::before {
  ...
  }
`;
```
```

File: packages/app-toolbox/src/Verify.tsx [hunk 3]
```diff
@@ -206,4 +208,17 @@ class Verify extends React.PureComponent<Props, State> {
...
 }
 
+const AlignedIcon = styled(Icon)`
+      &&&.ui--AlignedIcon::before {
+        position: relative;
+        left: 0.75rem;
+        top: 1rem; 
+        width: 32px;
+        height: 32px;
+        font-size: 24pt;
```
Comment (jacogr) [inline packages/app-toolbox/src/Verify.tsx]: Can we stick to `32px` here (i.e. no need to convert-in-the-head between the 2 measurements)

File: packages/app-toolbox/src/Verify.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (touhonoob) [reply packages/app-toolbox/src/Verify.tsx]: I moved the icon a bit to the right side. It should look better now.
Selection_162

File: packages/app-toolbox/src/Verify.tsx (hunk 2 above)
Comment (touhonoob) [reply packages/app-toolbox/src/Verify.tsx]: fixed

File: packages/app-toolbox/src/Verify.tsx (hunk 3 above)
Comment (touhonoob) [reply packages/app-toolbox/src/Verify.tsx]: fixed

**Comment by touhonoob** [comment]
@jacogr `&&&.ui--AlignedIcon i.big.icon` doesn't work as `&&&` here refers to the `i` element

**Comment by jacogr** [comment]
Good point, but we still need to apply it to the actual `i.big.icon` and not just the `::before` of the element.

**Comment by touhonoob** [comment]
@jacogr Got it. This should be fixed via 64c3f3eb3b47173145cf1f567d40e39c5007bf96

**Pull request merged**